tests/e2e/util.go

```diff
@@ -3152,3 +3153,26 @@ func statefulSetFromManifest(fileName string, ss *appsv1.StatefulSet) (*appsv1.S
 
 	return ss, nil
 }
+
+func waitForAllPodsToBeUpAndRunning(ctx context.Context, client clientset.Interface, namespace string) {
+	// If the status of the pods checked immediatly after the some update, they will be running
+	// wait for a min and run this method
+	time.Sleep(pollTimeoutShort)
```

Comment:
`wait.Poll` will wait for given time and then execute the test. we can avoid static sleep by using this method

Reply:
This is not as you see, once the rollout command returns success the waitforallpodstobeupandrunning method kickstarts and since all the csi pods will be still in running state (yet to take up the rolling update) the polling will not wait and returns success immediately. Removing the static sleep will not capture the pod status after the restart and any CLBO error if any.